tools-local/tasks/ValidateBinInspectResults.cs

```diff
@@ -52,8 +52,7 @@ public override bool Execute()
             {
                 foreach(var baselineFile in BaselineFiles)
                 {
-                    Regex ignoreRegEx = new Regex(baselineFile.ItemSpec);
-                    IEnumerable<XElement> baselineExcluded = errorRows.Where(f => ignoreRegEx.IsMatch(f.Descendants(s_NameElement).First().Value));
+                    IEnumerable<XElement> baselineExcluded = errorRows.Where(f => Regex.IsMatch(f.Descendants(s_NameElement).First().Value), baselineFile.ItemSpec, RegexOptions.IgnoreCase);
```

Comment:
Thanks, this is what I get for trying to write code from my phone when I'm half asleep.  I'll take a look.